web/controllers/homepage_controller.ex

```diff
@@ -4,6 +4,9 @@ defmodule App.HomepageController do
   alias App.{CMSRepo, Resources}
 
   @http Application.get_env(:app, :http)
+  @google_sheet_url_email Application.get_env(:app, :google_sheet_url_email)
+  @google_sheet_url_suggestion Application.get_env(
+    :app, :google_sheet_url_suggestion)
```

Comment:
It's because the line went over 80 chars and it wasn't working with:
```elixir
@google_sheet_url_suggestion
  Application.get_env(:app, :google_sheet_url_suggestion)
```